src/index.js

```diff
@@ -500,11 +500,25 @@ export const fuzzStaticMemberAssignmentTarget = f =>
 export const fuzzStaticMemberExpression = f =>
   ap(Shift.StaticMemberExpression, {"object": fuzzExpressionSuperProp, "property": fuzzIdentifierName}, f);
 
-export const fuzzStaticPropertyName = (f = new FuzzerState, {allowConstructor = true, allowPrototype = true} = {}) => { // todo avoid duplicate __proto__ simple properties
+export const fuzzStaticPropertyName = (f = new FuzzerState, {allowConstructor = true, allowPrototype = true} = {}) => {
+
+  let interestingNames = ['__proto__'];
```

Comment:
Surely there are more interesting names than this. `get`, `set`, `public`, `private`, `class`, `function`, `static`, `let` are just a few that come to mind.